lib/autoload/kuroko2/workflow/engine.rb

```diff
@@ -119,6 +120,7 @@ def process_next(node, token)
           token.mark_as_finished
           unless token.parent
             token.job_instance.touch(:finished_at)
+            Notifier.notify(:back_to_normal, token.job_instance) if token.context['RETRYING']
```

Comment:
Don't you need deleting the `RETRYING` context?

Reply:
`token` will be destroyed at [line 125](https://github.com/cookpad/kuroko2/blob/c5246f2fca1ca870a6ff5043768cb736fca8062b/lib/autoload/kuroko2/workflow/engine.rb#L125). So I decided there was no need to delete the `RETRYING` context.

Reply:
That's right (I forgot). That seems to be OK.

Reply:
Oh..., execuse me.
https://github.com/cookpad/kuroko2/pull/93/files#diff-f7b019ffd787516ae6ba13584edcd76dR121
It is necessary to put it outside the above conditions.